CONTRIBUTING.adoc

```diff
@@ -50,101 +45,98 @@ https://github.com/openshift/origin[origin repo].
 === Develop locally on your host
 
 You can develop OpenShift 3 on Windows, Mac, or Linux, but you'll need Docker
-installed on Linux to actually launch containers.
+installed on Linux to actually launch containers. Client and server binaries
+can be built locally or in the `openshift/release` container environemnt. The
+http://golang.org/[Go] programming language is only necessary for building on
+the local host.
```

Comment:
@adelton @stevekuznetsov @vbatts It is, but you need to be root user(or set pkgdir) as there is not yet *-race golang subpackage. I have only added it in f26 and up. I can add it to the f25 too, as I have been waiting for feedback on its addition. I haven't got any until now(do I understand that it "just" works for you?).

Bit on (un)related note, but is there way to disable run of all race,msan,... test by some env var/ flag to buildscripts? As these are not supported on all platforms/architectures.